src/code/z_player_lib.c

```diff
@@ -1800,49 +1986,396 @@ void Player_DrawZoraShield(PlayState* play, Player* player) {
+                    (player->skelAnime.animation == &gPlayerAnim_pz_wait)) {
+                    struct_80124618* phi_a0 =
+                        (player->skelAnime.animation == &gPlayerAnim_pz_wait)
+                            ? D_801C0608
+                            : ((player->skelAnime.animation == &gPlayerAnim_pg_wait)
+                                   ? D_801C0628
+                                   : ((player->skelAnime.animation == &gPlayerAnim_pg_punchC) ? D_801C07C0
+                                                                                              : D_801C0460));
+
+                    func_80124618(phi_a0, player->skelAnime.curFrame, player->unk_AF0);
+                } else {
+                    player->unk_AF0[0].y = 1.0f;
+                }
+
+                if (player->transformation == PLAYER_FORM_ZORA) {
+                    player->unk_AF0[0].y *= 1.0f + (0.29999995f * player->unk_B10[0]);
+                    player->unk_AF0[0].x = 1.0f;
+                    player->unk_AF0[0].z = 1.0f;
+                }
+                Matrix_Scale(player->unk_AF0[0].x, player->unk_AF0[0].y, player->unk_AF0[0].z, MTXMODE_APPLY);
+            }
+        } else if (limbIndex == PLAYER_LIMB_UPPER_ROOT) {
+            s16 xRot;
+            s16 zRot = 0x44C;
+
+            if (player->unk_AA8 != 0) {
+                Matrix_RotateZS(zRot, MTXMODE_APPLY);
+                Matrix_RotateYS(player->unk_AA8, MTXMODE_APPLY);
+            }
+
+            if (player->unk_AB2.y != 0) {
+                Matrix_RotateYS(player->unk_AB2.y, MTXMODE_APPLY);
+            }
+
+            xRot = player->unk_AB2.x;
+            if ((player->transformation == PLAYER_FORM_DEKU) && (player->stateFlags3 & PLAYER_STATE3_40)) {
+                if (player->heldActor != NULL) {
+                    xRot += (s16)(((EnArrow*)(player->heldActor))->bubble.unk_144 * -470.0f);
+                }
+            }
+
+            Matrix_RotateXS(xRot, MTXMODE_APPLY);
+            if (player->unk_AB2.z != 0) {
+                Matrix_RotateZS(player->unk_AB2.z, MTXMODE_APPLY);
+            }
+        } else {
+            func_80125500(play, player, limbIndex, pos, rot);
+        }
+    }
+
+    return false;
+}
+
+void func_80125CE0(Player* player, struct_80124618* arg1, Vec3f* pos, Vec3s* rot) {
+    Matrix_TranslateRotateZYX(pos, rot);
+    func_80125318(pos, rot);
+    func_80124618(arg1, player->skelAnime.curFrame, player->unk_AF0);
+    Matrix_Scale(player->unk_AF0[0].x, player->unk_AF0[0].y, player->unk_AF0[0].z, MTXMODE_APPLY);
+}
+
+// Player_OverrideLimbDrawGameplayDefault
+s32 func_80125D4C(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* actor) {
+    Player* player = (Player*)actor;
+
+    if (!Player_OverrideLimbDrawGameplayCommon(play, limbIndex, dList, pos, rot, &player->actor)) {
+        if (limbIndex == PLAYER_LIMB_LEFT_HAND) {
+            Gfx** var_a0 = player->leftHandDLists;
```

Comment:
```suggestion
            Gfx** leftHandDLists = player->leftHandDLists;
```
Same for `rightHandDLists` and `sheathDLists`